src/locales/en-US/commands.json

```diff
@@ -42,10 +42,70 @@
     "commandDescription": "Checks if the bot is online"
   },
   "spotify": {
-    "commandDescription": "Searches for a song on Spotify",
-    "commandUsage": "<track name>",
+    "commandDescription": "Searches for a song, album, artist or a playlist and displays an user info on Spotify",
+    "commandUsage": "<type> <query>",
     "noTracksFound": "I couldn't find any tracks with that name.",
-    "noTrackName": "You need to give me a track name."
+    "noType": "Please insert a valid type",
+    "types": "Types",
+    "album": "Album",
+    "genres": "Genres",
+    "artist": "Artist",
+    "artistPlural": "Artists",
+    "track": "Track",
+    "trackPlural": "Tracks",
+    "albumPlural": "Albums",
+    "moreTracks": "...and more {{tracks}} tracks",
+    "moreAlbums": "...and more {{albums}} albums",
+    "followers": "Followers",
+    "followersCount": "{{followers}} followers",
+    "selectResult": "Please send the number of the result (10 seconds)",
```

Comment:
Could we get some context, please? It's rather hard to understand.

Reply:
done